fetcher/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Fetcher
+
+## Requirements
+
+- Node 6+
+
+## Install
+
+    # create a PostgreSQL database
+    createdb sm-fetcher
+    # create tables
+    psql -d sm-fetcher -f ../db/init-schema.sql
+    # install dependencies
+    npm install
+    # start web server
+    npm run dev
+    
+## Usage
+
+Use `curl` or `postman` or similar for the following:
+
+1. [Generate a GUID](https://www.guidgen.com/)
+2. [Create a Twitter app](https://apps.twitter.com/) or use an exiting one
+3. `POST http://locahlost:8081/registration/register` with `{"applicationId":"<YOUR GUID>"}`
+4. `POST http://locahlost:8081/twitter/register` with `{"applicationId":"<YOUR GUID>", "twitterId":"<TWITTER CONSUMER KEY>", "twitterSecret":"<TWITTER CONSUMER SECRET>"}`
+5. ???
```

Comment:
This is done by the webserver (except for registering a twitter app of course). You can see how its done in the `/register` route of the webserver. https://github.com/jannikko/social-monitor/blob/master/web/server/services/ai.js In the `registerApplication` function you can see how the application is registered with the ai and the fetcher service. There is a route for this on the frontend aswell it should be `/register`. All of this was done at the end of the project and is pretty messy.